### ISSUE TITLE
作者考虑支持最近的FlowOPD吗🤔

### ISSUE THREAD

**Issue by collectionlora** [X-GenGroup/Flow-Factory#166]

**Comment by Jayce-Ping** [comment]
那篇文章的正确性有待考证。他的codebase少了一个`advanatges.detach()`导致实际work的是直接蒸馏的loss而不是文章中声称的policy-gradient. 加上一行`advantages.detach()`就不涨点了. 我目前测出来的结论也是直接policy-gradient（以及各种小魔改）是不work的，理论上也是期望为0. 更多可以看DiffusionOPD这篇文章: https://arxiv.org/abs/2605.15055. 这篇文章中提到的算法我会考虑在后续支持，但需要等作者本人放出代码之后并对齐验证之后了。

**Comment by Jayce-Ping** [comment]
[Diffusion-OPD](https://github.com/ali-vilab/DiffusionOPD/) is supported in the main branch now. Check PR #170 for more info.
